### ISSUE TITLE
Enhancement: What the flag?

### ISSUE THREAD

**Issue by develCuy** [steemit/steemit.com#186]
![What the flag?](https://pbs.twimg.com/media/CqvCnWLXYAAj7HA.jpg:large)

The flag feature needs improvement. We can't simply let people do it straight, even asking them about the damage it would cause, which is actually a way to encourage aggressiveness.
Let's suppose I find a post that is disgusting for me, because it goes completely against my believes, then I notice it got tens of upvotes and lots of comments. So I see that flag button, press it and what I read is:

_"Flagging a post can remove rewards and make this material less visible. You can still unflag or upvote later if you change your mind"_

Sweet! I find it perfect for revenge!

In short, current implementation promotes abuse, this language needs to be improved and perhaps there would be some categories just like in facebook:

- It's annoying or not interesting
- I think it shouldn't be on Steemit
- It's SPAM

Other downside is moderation, If we are letting people flag like crazy, the moderation queue would get exploited, everyone looses!

**Comment by jcalfee** [comment]
This flag is for the following:

* Spam (grossly miss-categorized content)
* Fraud or Plagerizm
* Hate Speach

**Comment by jcalfee** [comment]
Flagging is for:
* Fraud or Plagiarism
* Hate Speech or Internet Trolling
* Intentional miss-categorized content or Spam

For a difference of opinion or offensive material please use the comment section and seek support from your peers.

**Comment by develCuy** [comment]
that was a fast. Congratulations!

**Comment by arhag** [comment]
The last item should read:
Intentionally miscategorized content or Spam
